assets/js/components/labels/LabelShow.jsx

```diff
@@ -126,24 +126,17 @@ class LabelShow extends Component {
     this.props.updateLabelNotificationWebhooks(webhooks);
   }
 
-  handleToggleDebug = () => {
-    const { showDebugSidebar } = this.state
-
-    if (!showDebugSidebar) {
-      this.props.toggleLabelDebug(this.props.match.params.id)
-      analyticsLogger.logEvent("ACTION_OPEN_LABEL_DEBUG", { "id": this.props.match.params.id })
```

Comment:
wouldn't we wanna keep the analytics log?

Reply:
good catch ty